### PR TITLE
AEX-5 - Final

### PR DESCRIPTION
**Pull request by shekhar-shubhendu**

### CODE REVIEW

**Comment by shekhar-shubhendu** [comment]
closes #22

File: AEXS/aex-5.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Title: Inter-Wallet Communication
 Author: Shubhendu Shekhar <@shekhar-shubhendu>
 License: BSD-3-Clause
 Discussions-To: https://forum.aeternity.com/t/aex-5-inter-wallet-communication/3371
-Status: Draft
+Status: Last call (2019-05-29 to 2019-06-04)
```
Comment (davidyuk) [inline AEXS/aex-5.md]: Shouldn't it be changed to "Review" firstly?

Comment (shekhar-shubhendu) [reply AEXS/aex-5.md]: we've been through various discussions and I guess we can say it has been through review and lot of scrutiny. moving to review is only for symbolic reasons then I think we should skip it and save some time and money.

Comment (davidyuk) [reply AEXS/aex-5.md]: > Each proposal must start in the Draft state and will move through the phases in accordance to the criteria defined in this document.

https://github.com/aeternity/AEXs/blob/master/AEXS/aex-1.md#preamble

Comment (thepiwo) [reply AEXS/aex-5.md]: @davidyuk I think just the representation of these stages was missing for this one. `Active Review` was done in various discussions and calls, but was forgotten to update in the repository.

**Review by thepiwo — APPROVED** [review]

**Review by davidyuk — CHANGES_REQUESTED** [review]
I think Final status should be delayed until corresponding issues get resolved:
https://github.com/aeternity/AEXs/issues/42
https://github.com/aeternity/AEXs/issues/26
Their resolving may lead to breaking changes. Also, there are some issues related to AEX-2 that are connected to this one, because it reuses some stuff from there.

**Comment by thepiwo** [comment]
Thank you @davidyuk for all the feedback and discussion points. I do agree with you that we will have to delay a final version due to this, same as AEX-2.

**Pull request closed without merging**